var/spack/repos/builtin/packages/r-ade4/package.py

```diff
@@ -11,9 +11,12 @@ class RAde4(RPackage):
     Environmental Sciences"""
 
     homepage = "http://pbil.univ-lyon1.fr/ADE-4"
-    url      = "https://cran.r-project.org/src/contrib/ade4_1.7-6.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/ade4"
+    url      = "https://cloud.r-project.org/src/contrib/ade4_1.7-6.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/ade4"
 
+    version('1.7-13', sha256='f5d0a7356ae63f82d3adb481a39007e7b0d70211b8724aa686af0c89c994e99b')
+    version('1.7-11', sha256='4ccd799ae99bd625840b866a697c4a48adb751660470bf0d6cf9207b1927a572')
     version('1.7-6', '63401ca369677538c96c3d7b75b3f4a1')
 
     depends_on('r@2.10:')
```

Comment:
Probably `type=('build', 'run')`

Reply:
Since spack does not have a version of R < 3.1 I think I will remove any specs < 3.1.2. For the ones above that I will add the type.

Reply:
I like keeping the constraint just in case someone adds an old version, or more importantly, if someone uses a system installation as an external package and it is old.

Reply:
Got it.

Reply:
This is fixed with the latest commit.